### DISCUSSION TITLE
Setting up cars in AirSim with street view

### DISCUSSION THREAD

**Discussion by kulkarnikeerti** [microsoft/AirSim#4080] [Support Q&A] [answered]
I have everything running without any issues with AirSim and Unreal Engine. 

I am also able to run 
`python hello_car.py`

And the editor looks something like below
![image](https://user-images.githubusercontent.com/12002873/137154735-e96b9ba3-b968-466a-90ec-722f4eaab3c6.png)

But I would like to setup something like in this video with Street view
[https://www.youtube.com/watch?v=gnz1X3UNM5Y](url)

Is there a setup guideline for the same? or am I just missing something here?

Thanks in advance

**Comment by jonyMarino** [comment, marked as answer]
@kulkarnikeerti, those environments are from the AirSim releases:
https://github.com/microsoft/AirSim/releases/tag/v1.6.0-linux
I see  he was using City, Neighborhood, and Landscape
